// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots: onFinish assembly/jest-reporter.ts 1`] = `
"
   [Error]: ReflectedValue Cannot log ReflectedValue of id -1. Index out of bounds.
   [Stack]:     at start:assembly/jest-reporter-fail~anonymous|0~anonymous|3 (wasm-function[63]:0x1707)
               at ../assembly/assembly/internal/call/__call (wasm-function[128]:0x33da)
    [File]: assembly/jest-reporter.ts RTrace: +10
  [Groups]: 2 pass, 2 total
  [Result]: ✖ FAIL
[Snapshot]: 0 total, 0 added, 0 removed, 0 different
 [Summary]: 4 pass,  5 fail, 9 total
    [Time]: 0ms

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

"
`;

exports[`snapshots: onGroupEnd  1`] = `""`;

exports[`snapshots: onGroupEnd examples 1`] = `
"    [Todo]: example todo

"
`;

exports[`snapshots: onGroupStart examples 1`] = `
"[Describe]: examples

     [Log]: \\"testing in a group\\"
   [Stack]: at ../assembly/assembly/internal/Reflect/Reflect.attachStackTrace (wasm-function[41]:0x11e7)
            at ../assembly/assembly/internal/log/log<~lib/string/String> (wasm-function[61]:0x16f0)
            at start:assembly/jest-reporter-fail~anonymous|0 (wasm-function[125]:0x33ac)
            at ../assembly/assembly/internal/call/__call (wasm-function[128]:0x33da)
"
`;

exports[`snapshots: onTestFinish examples example 1 1`] = `
" [Success]: ✔ example 1
"
`;

exports[`snapshots: onTestFinish examples example 1 2`] = `
" [Success]: ✔ example 1
"
`;

exports[`snapshots: onTestFinish examples log value 1`] = `
" [Success]: ✔ log value
     [Log]: \\"testing in a test\\"
   [Stack]: at ../assembly/assembly/internal/Reflect/Reflect.attachStackTrace (wasm-function[41]:0x11e7)
            at ../assembly/assembly/internal/log/log<~lib/string/String> (wasm-function[61]:0x16f0)
            at start:assembly/jest-reporter-fail~anonymous|0~anonymous|2 (wasm-function[62]:0x1700)
            at ../assembly/assembly/internal/call/__call (wasm-function[128]:0x33da)
"
`;

exports[`snapshots: onTestFinish examples push an error to the error array 1`] = `
"    [Fail]: ✖ push an error to the error array
"
`;

exports[`snapshots: onTestFinish examples something fails 1`] = `
"    [Fail]: ✖ something fails
  [Actual]: -1
[Expected]: 42
 [Message]: Here is an example failure message.
   [Stack]: RuntimeError: unreachable
            at ../assembly/assembly/internal/assert/assert (wasm-function[44]:0x1244)
            at ../assembly/assembly/internal/Expectation/Expectation<i32>#toBe (wasm-function[47]:0x128c)
            at start:assembly/jest-reporter-fail~anonymous|0~anonymous|4 (wasm-function[64]:0x1719)
            at ../assembly/assembly/internal/call/__call (wasm-function[128]:0x33da)
"
`;

exports[`snapshots: onTestFinish examples something negated fails 1`] = `
"    [Fail]: ✖ something negated fails
  [Actual]: 42
[Expected]: Not 42
 [Message]: 42 is 42
   [Stack]: RuntimeError: unreachable
            at ../assembly/assembly/internal/assert/assert (wasm-function[44]:0x1244)
            at ../assembly/assembly/internal/Expectation/Expectation<i32>#toBe (wasm-function[47]:0x128c)
            at start:assembly/jest-reporter-fail~anonymous|0~anonymous|7 (wasm-function[100]:0x2b01)
            at ../assembly/assembly/internal/call/__call (wasm-function[128]:0x33da)
     [Log]: \\"another string\\"
   [Stack]: at ../assembly/assembly/internal/Reflect/Reflect.attachStackTrace (wasm-function[41]:0x11e7)
            at ../assembly/assembly/internal/log/log<~lib/string/String> (wasm-function[61]:0x16f0)
            at start:assembly/jest-reporter-fail~anonymous|0~anonymous|7 (wasm-function[100]:0x2af0)
            at ../assembly/assembly/internal/call/__call (wasm-function[128]:0x33da)
"
`;

exports[`snapshots: onTestFinish examples something using toBe fails with a reference that is similar 1`] = `
"    [Fail]: ✖ something using toBe fails with a reference that is similar
  [Actual]: Vec3 {
      x: 1.0 as f32,
      y: 2.0 as f32,
      z: 3.0 as f32
  }
[Expected]: \\"Serializes to same value.\\"
 [Message]: not the same pointer
   [Stack]: RuntimeError: unreachable
            at ../assembly/assembly/internal/assert/assert (wasm-function[44]:0x1244)
            at ../assembly/assembly/internal/Expectation/Expectation<assembly/jest-reporter-fail/Vec3>#toBe (wasm-function[122]:0x32f6)
            at start:assembly/jest-reporter-fail~anonymous|0~anonymous|8 (wasm-function[123]:0x333e)
            at ../assembly/assembly/internal/call/__call (wasm-function[128]:0x33da)
"
`;

exports[`snapshots: onTestFinish examples this should throw but it does not 1`] = `
"    [Fail]: ✖ this should throw but it does not
 [Message]: it should throw
"
`;

exports[`snapshots: onTestFinish examples when two things should equal each other 1`] = `
"  [Throws]: ✔ when two things should equal each other RTrace: +5
"
`;

exports[`snapshots: onTestStart examples example 1 1`] = `""`;

exports[`snapshots: onTestStart examples example 1 2`] = `""`;

exports[`snapshots: onTestStart examples log value 1`] = `""`;

exports[`snapshots: onTestStart examples push an error to the error array 1`] = `""`;

exports[`snapshots: onTestStart examples something fails 1`] = `""`;

exports[`snapshots: onTestStart examples something negated fails 1`] = `""`;

exports[`snapshots: onTestStart examples something using toBe fails with a reference that is similar 1`] = `""`;

exports[`snapshots: onTestStart examples this should throw but it does not 1`] = `""`;

exports[`snapshots: onTestStart examples when two things should equal each other 1`] = `""`;
